recipe/meta.yaml

```diff
@@ -50,6 +55,15 @@ requirements:
     - googledrivedownloader
     - jinja2
     - pyparsing
+    # Workaround to add `libxcrypt` for old Pythons
+    # xref: https://github.com/conda-forge/python-feedstock/pull/668#discussion_r1523986176
+    - libxcrypt  # [linux and py<39]
```

Comment:
Looks like this doesn't help. Let's drop it

As this only affects Python 3.8 and it's pretty old at this point, maybe we should consider just adding a `skip` for Python 3.8

```suggestion
```